powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthIdentityVerificationTest.java

```diff
@@ -1069,6 +1074,23 @@ private void initPresenceCheck(String processId) throws Exception {
         assertEquals(200, stepLogger.getResponse().getStatusCode());
     }
 
+    private void submitPresenceCheck(final String processId) throws Exception {
+        if (config.isSkipPresenceCheck()) {
+            return;
+        }
+        // TODO (racansky, 2022-09-26) change to PresenceCheckSubmitRequest
+        final PresenceCheckInitRequest presenceCheckRequest = new PresenceCheckInitRequest();
```

Comment:
Can we fix this TODO?

Reply:
Yes, but you can prepare it already, I think.

Reply:
Ok, let's wait with merge of enrollment-server after testing app with iProov and then we may fix it.